### DISCUSSION TITLE
ui.persona is awesome in the new release -- thanks

### DISCUSSION THREAD

**Discussion by sairaamv** [h2oai/wave#1009] [General]
Hi Wave team, ui.persona is awesome. Bet it is better at anytime than the `Account` fluent UI icon 

Thanks.

**Comment by mturoci** [comment]
Glad you like it :)

**Comment by sairaamv** [comment]
Would it be a good option to add pixel value support as well? 

thanks

**Reply by mturoci** [reply]
We try to avoid supporting arbitrary values when possible in order to ensure consistency across Wave apps. For this case, you can use `size` attr with allowed values `'xl' | 'l' | 'm' | 's' | 'xs'`.
